index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <nav class="toolbar">
+        <h2>Welcome</h2>
+        <button class="icon">☰</button>
+    </nav>
+    <main>
+        <section class="header">
+            <h2 class="name">Hey there. I'm Juan Sebastian</h2>
+            <h2 class="career">I'm a software developer</h2>
+            <p class="paragraph"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </section>
+        <section class="social">
+            <a class="github" href="https://github.com/Juanse7793"><img src="/images/github.svg" alt="github"></a>
+            <a class="linkedin" href="https://www.linkedin.com/in/juan-sebastian-sotomayor-2bb395198/"><img src="/images/Linkedin_icon.svg" alt="linkedin"></a>
+            <a class="twitter" href="https://twitter.com/Juanse77930"><img src="/images/twitter.svg" alt="twitter"></a>
```

Comment:
Due to semantic reasons, it's a good practice to wrap these social icons inside an unordered list `ul` tag 🤓
Kindly perform these suggestions so that the list of icons uses a ul tag to group them, and li per icon!